datahub-web-react/src/app/permissions/policy/PolicyActorForm.tsx

```diff
@@ -46,12 +52,39 @@ export default function PolicyActorForm({ policyType, actors, setActors }: Props
     // Search for actors while building policy.
     const [userSearch, { data: userSearchData }] = useGetSearchResultsLazyQuery();
     const [groupSearch, { data: groupSearchData }] = useGetSearchResultsLazyQuery();
-
+    const { data: ownershipData } = useListOwnershipTypesQuery({
+        variables: {
+            input: {},
+        },
+    });
+    const ownershipTypes =
+        ownershipData?.listOwnershipTypes?.ownershipTypes.filter((type) => type.urn !== 'urn:li:ownershipType:none') ||
+        [];
+    const ownershipTypesMap = Object.fromEntries(ownershipTypes.map((type) => [type.urn, type.info?.name]));
     // Toggle the "Owners" switch
     const onToggleAppliesToOwners = (value: boolean) => {
         setActors({
             ...actors,
             resourceOwners: value,
+            resourceOwnersTypesUrns: value ? actors.resourceOwnersTypesUrns : null,
```

Comment:
Same as above - we can remove the "urns" suffix in all of these names

Reply:
Done